conf/site.conf.sample

```diff
@@ -27,8 +27,9 @@ YOE_PROFILE = "yoe-musl-sysvinit-eglfs"
 YOE_PROFILE = "yoe-musl-sysvinit-wayland"
 YOE_PROFILE = "yoe-musl-sysvinit-x11"
 # configure docker container to run bitbake in
-export DOCKER_REPO=yoedistro/yoe-build:bullseye
-
+export DOCKER_REPO=yoedistro/yoe-build:bookworm-x86_64
```

Comment:
Do you think we should note the bookworm change in the changelog?

Reply:
I will add in followup